drivers/builtin/src/aes.c

```diff
@@ -480,6 +480,36 @@ void mbedtls_aes_free(mbedtls_aes_context *ctx)
     mbedtls_platform_zeroize(ctx, sizeof(mbedtls_aes_context));
 }
 
+void mbedtls_aes_get_implementation(mbedtls_aes_implementation **aes_imp_out)
```

Comment:
Since this is `void` anyway, it would be neater to have this return the `mbedtls_aes_implementation` rather than having an out parameter. This would also enable you to replace the `goto`s with `return`s.